sys/linux/socket_netlink_route.txt

```diff
@@ -758,6 +799,53 @@ ifa_cacheinfo {
 	tstamp		int32
 }
 
+br_vlan_db_dump_pol [
+	BRIDGE_VLANDB_DUMP_FLAGS	nlattr[BRIDGE_VLANDB_DUMP_FLAGS, flags[br_vlan_dumpflags, int32]]
+]
+
+br_vlan_db_policy [
+	BRIDGE_VLANDB_ENTRY	nlnest[BRIDGE_VLANDB_ENTRY, br_vlan_db_policy_entry]
+] [varlen]
+
+br_vlan_db_policy_entry [
+	BRIDGE_VLANDB_ENTRY_INFO	nlattr[BRIDGE_VLANDB_ENTRY_INFO, bridge_vlan_info]
+	BRIDGE_VLANDB_ENTRY_RANGE	nlattr[BRIDGE_VLANDB_ENTRY_RANGE, int16[BRIDGE_VLAN_INFO_RANGE_BEGIN:BRIDGE_VLAN_INFO_RANGE_END]]
+	BRIDGE_VLANDB_ENTRY_STATE	nlattr[BRIDGE_VLANDB_ENTRY_STATE, flags[br_vlan_state_options, int8]]
+	BRIDGE_VLANDB_ENTRY_TUNNEL_INFO	nlnest[BRIDGE_VLANDB_ENTRY_TUNNEL_INFO, array[br_vlandb_tinfo_pol]]
```

Comment:
Are you sure there's an `array` here? I'm failing to understand that from the code.

Reply:
No. It seems to me it should be 
`+ ...T_INFO	nlnest[...T_INFO, br_vlandb_tinfo_pol]`
instead (removing the array part). It is simply nested.
Changed locally, will check coverage first.

Reply:
Pushed the update.